pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/FunctionMetaDataManager.java

```diff
@@ -52,20 +61,15 @@
     @VisibleForTesting
     final Map<String, Map<String, Map<String, FunctionMetaData>>> functionMetaDataMap = new ConcurrentHashMap<>();
 
-    // A map in which the key is the service request id and value is the service request
-    private final Map<String, ServiceRequestInfo> pendingServiceRequests = new ConcurrentHashMap<>();
-
     private final ServiceRequestManager serviceRequestManager;
     private final SchedulerManager schedulerManager;
     private final WorkerConfig workerConfig;
     private final PulsarClient pulsarClient;
     private final ErrorNotifier errorNotifier;
 
     private FunctionMetaDataTopicTailer functionMetaDataTopicTailer;
-
-    @Setter
-    @Getter
-    boolean isInitializePhase = false;
```

Comment:
Why are we getting rid of the "isInitializePhase" flag?  The flag is in place so we don't call schedule() for every message during the initialization phase and we wait to just call schedule at the end of the init phase

Reply:
I see in "processRequest()" no scheduling is called